linode_api4/objects/object_storage.py

```diff
@@ -31,10 +31,10 @@ class ObjectStorageBucket(DerivedBase):
     id_attribute = "label"
 
     properties = {
-        "cluster": Property(),
+        "cluster": Property(identifier=True),
         "created": Property(is_datetime=True),
         "hostname": Property(),
-        "label": Property(),
+        "label": Property(identifier=True),
```

Comment:
These weren't flagged as IDs so they were always overridden after the above change. This should fix that up 🙂